build-scripts/hack/sync-registry.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+DIR=$(realpath $(dirname "${0}"))
+
+docker run -v "${DIR}/registry-k8s-io.yaml":/config.yaml quay.io/skopeo/stable:v1.15 sync \
```

Comment:
where is `registry-k8s-io.yaml`?

Reply:
I accepted your github suggestion and missed that.